deploy-nebula-graph-with-docker-and-compose.md

```diff
@@ -0,0 +1,173 @@
+# Deploying Nebula Graph Cluster with Docker and Docker Compose
+
+## Overview
+
+In this document, we will walk you through the process of deploying a **Nebula Graph** cluster with [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/). We will also show you how to [check the services status of **Nebula Graph**](#checking-the-status-and-ports-of-nebula-graph-services), how to [check the cluster data and logs](#checking-the-cluster-data-and-logs), and how to [stop the services of **Nebula Graph**](#stopping-the-services-of-nebula-graph).
+
+## Prerequisite
+
+Before you start deploying the **Nebula Graph** cluster, ensure that you have installed the latest version of [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/).
+
+## Services of Nebula Graph to Be Deployed
+
+In this guide, we are going to deploy the following services of **Nebula Graph**:
+
+* 3 replicas of nebula-metad service
+* 3 replicas of nebula-storaged service
+* 1 replica of nebula-graphd service
+
+## Deploying the Nebula Graph Cluster
+
+You can deploy the **Nebula Graph** cluster by the following steps:
+
+1. Clone the `nebula-docker-compose` repository to your local computer.
+
+```shell
+$ git clone https://github.com/vesoft-inc/nebula-docker-compose.git
+```
+
+2. Change your current directory to the `nebula-docker-compose` directory.
+
+```shell
+$ cd nebula-docker-compose/
+```
+
+3. Start all the services of **Nebula Graph**.
+
+```shell
+$ sudo docker-compose up -d
```

Comment:
Needn't `sudo`

Reply:
If you do not have the root privilege, sudo is needed